examples/x86_64.swiftinterface

```diff
@@ -0,0 +1,511 @@
+  public var participantsAdded: [AzureCommunicationChat.SignalingChatParticipant]?
+  public var addedBy: AzureCommunicationChat.SignalingChatParticipant?
+  @objc deinit
+}
+@_hasMissingDesignatedInitializers public class ParticipantsRemovedEvent : AzureCommunicationChat.BaseChatThreadEvent {
+  public var removedOn: AzureCore.Iso8601Date?
+  public var participantsRemoved: [AzureCommunicationChat.SignalingChatParticipant]?
+  public var removedBy: AzureCommunicationChat.SignalingChatParticipant?
+  @objc deinit
+}
+public enum ChatEventId : Swift.String {
+  case chatMessageReceived
+  case typingIndicatorReceived
+  case readReceiptReceived
+  case chatMessageEdited
+  case chatMessageDeleted
+  case chatThreadCreated
+  case chatThreadPropertiesUpdated
+  case chatThreadDeleted
+  case participantsAdded
+  case participantsRemoved
+  public typealias RawValue = Swift.String
+  public init?(rawValue: Swift.String)
+  public var rawValue: Swift.String {
+    get
+  }
+}
+public struct ChatMessagesCollection : Swift.Codable {
+  public let value: [AzureCommunicationChat.ChatMessage]
+  public let nextLink: Swift.String?
+  public init(value: [AzureCommunicationChat.ChatMessage], nextLink: Swift.String? = nil)
+  public init(from decoder: Swift.Decoder) throws
+  public func encode(to encoder: Swift.Encoder) throws
+}
+public struct ChatParticipantsCollection : Swift.Codable {
+  public let value: [AzureCommunicationChat.ChatParticipant]
+  public let nextLink: Swift.String?
+  public init(value: [AzureCommunicationChat.ChatParticipant], nextLink: Swift.String? = nil)
+  public init(from decoder: Swift.Decoder) throws
+  public func encode(to encoder: Swift.Encoder) throws
+}
+public struct SendReadReceiptRequest : Swift.Codable {
+  public let chatMessageId: Swift.String
+  public init(chatMessageId: Swift.String)
+  public init(from decoder: Swift.Decoder) throws
+  public func encode(to encoder: Swift.Encoder) throws
+}
+public struct SendTypingNotificationOptions : AzureCore.RequestOptions {
+  public let clientRequestId: Swift.String?
+  public let cancellationToken: AzureCore.CancellationToken?
+  public var dispatchQueue: Dispatch.DispatchQueue?
+  public var context: AzureCore.PipelineContext?
+  public init(clientRequestId: Swift.String? = nil, cancellationToken: AzureCore.CancellationToken? = nil, dispatchQueue: Dispatch.DispatchQueue? = nil, context: AzureCore.PipelineContext? = nil)
+}
+public struct GetChatMessageOptions : AzureCore.RequestOptions {
+  public let clientRequestId: Swift.String?
+  public let cancellationToken: AzureCore.CancellationToken?
+  public var dispatchQueue: Dispatch.DispatchQueue?
+  public var context: AzureCore.PipelineContext?
+  public init(clientRequestId: Swift.String? = nil, cancellationToken: AzureCore.CancellationToken? = nil, dispatchQueue: Dispatch.DispatchQueue? = nil, context: AzureCore.PipelineContext? = nil)
+}
+public struct SendChatReadReceiptOptions : AzureCore.RequestOptions {
```

Comment:
Should this be consistent with `SendReadReceiptRequest`, i.e. `SendReadReceiptOptions`?